include/ttmlir/Dialect/D2M/Analysis/DstAnalysis.h

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: (c) 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TTMLIR_DIALECT_D2M_ANALYSIS_DSTREQUIREMENTANALYSIS_H
+#define TTMLIR_DIALECT_D2M_ANALYSIS_DSTREQUIREMENTANALYSIS_H
```

Comment:
:warning: **llvm\-header\-guard** :warning:
header guard does not follow preferred style
```suggestion
#ifndef TTMLIR_DIALECT_D2M_ANALYSIS_DSTANALYSIS_H
#define TTMLIR_DIALECT_D2M_ANALYSIS_DSTANALYSIS_H
```